src/main/java/hw_9/task1/Planet.java

```diff
@@ -0,0 +1,12 @@
+package hw_9.task1;
+
+public interface Planet {
+
+    final String fromSun = " from the SUN: ";
+
+    default void displayInfo() {
+        System.out.println(this.getClass().toString());
```

Comment:
this sout display not very useful information
![Screenshot_6](https://github.com/user-attachments/assets/4471f50d-80b2-4c2f-8fcc-74ce34cfe63b)

So, i guest i guess we don't need to do it as default, just - void displayInfo();